src/common/organizationCustomerDetails/OrganizationCustomerDetails.tsx

```diff
@@ -28,6 +28,7 @@ export interface OrganizationCustomerDetailsProps {
   } | null;
   primaryEmail?: string | null;
   primaryPhone?: string | null;
+  language: Language | null;
```

Comment:
What's the reason of not using `BerthApplicationLanguage` instead of `Language` and mapping the values?

Reply:
It is so that language information can be shown in customer details on both customer and berth application views, and later in the winter storage application view.

`Language` is used on customer profiles and `BerthApplicationLanguage` is used on berth applications, but `Language` is the more generic sounding of the two.